app/db/queries.ts

```diff
@@ -1,12 +1,22 @@
-import { sql } from '@vercel/postgres';
-import { toTitleCase } from './utils';
+import { createPool } from "@vercel/postgres";
+import { toTitleCase } from "./utils";
 
 type Folder = {
   name: string;
   email_count: string;
 };
 
+const connectionString = process.env.DATABASE_URL || "";
+
+type Primitive = string | number | boolean | undefined | null;
+
+const sql = async (q: TemplateStringsArray, ...values: Primitive[]) => {
+  const client = await createPool({ connectionString });
+  return client.sql(q, ...values);
+};
```

Comment:
even overriding this throws an error
<img width="444" alt="image" src="https://github.com/kiwicopple/nextjs-postgres-email-client/assets/10214025/72f6f318-f4a1-47fb-a2d9-67db38f8b9a6">